Author: Jamie Strandboge <jamie@canonical.com>
Description: always install python modules in the proper location when creating
 deb files
Forwarded: no (Debian/Ubuntu specific)

---
 libraries/libapparmor/swig/python/Makefile.am |    2 +-
 utils/Makefile                                |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: b/libraries/libapparmor/swig/python/Makefile.am
===================================================================
--- a/libraries/libapparmor/swig/python/Makefile.am
+++ b/libraries/libapparmor/swig/python/Makefile.am
@@ -15,7 +15,7 @@ all-local: libapparmor_wrap.c setup.py
 	$(PYTHON) setup.py build
 
 install-exec-local:
-	$(PYTHON) setup.py install --root="/$(DESTDIR)" --prefix="$(prefix)"
+	$(PYTHON) setup.py install --root="/$(DESTDIR)" --prefix="$(prefix)" --install-layout=deb
 
 clean-local:
 	if test -x "$(PYTHON)"; then $(PYTHON) setup.py clean; fi
Index: b/utils/Makefile
===================================================================
--- a/utils/Makefile
+++ b/utils/Makefile
@@ -57,7 +57,7 @@ install: ${MANPAGES} ${HTMLMANPAGES}
 	$(MAKE) install_manpages DESTDIR=${DESTDIR}
 	$(MAKE) -C vim install DESTDIR=${DESTDIR}
 	ln -sf aa-status.8 ${DESTDIR}/${MANDIR}/man8/apparmor_status.8
-	${PYTHON} ${PYSETUP} install --prefix=${PYPREFIX} --root=${DESTDIR} --version=${VERSION}
+	${PYTHON} ${PYSETUP} install --prefix=${PYPREFIX} --root=${DESTDIR} --install-layout=deb --version=${VERSION}
 
 .PHONY: clean
 ifndef VERBOSE
